lib/site-inspector/checks/accessibility.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+class SiteInspector
+  class Endpoint
+    class Accessibility < Check
+      
+      def section508
+        pa11y('Section508')
+      end
+      
+      def wcag2a
+        pa11y('WCAG2A')
+      end
+      
+      def wcag2aa
+        pa11y('WCAG2AA')
+      end
+      
+      def wcag2aaa
+        pa11y('WCAG2AAA')
+      end
+      
+      private
+      
+      def pa11y(reporter)
+        json_string = `pa11y https://18f.gsa.gov -s #{reporter} -r json`
```

Comment:
I'd also add a `pa11y_installed?` or similar method, which would check that `pa11y` is installed and raise an appropriate error if not. Also perhaps a `pa11y_version` method, to expose that programmatically (which itself, could be the `pa11y_installed` check)?